src/server/src/controllers/comment.controllers.ts

```diff
@@ -0,0 +1,108 @@
+import { Request, Response } from 'express';
+import Author from '../models/Author';
+import Comment from '../models/Comment';
+import Post from '../models/Post';
+import { AuthenticatedRequest } from '../types/auth';
+import { PaginationRequest } from '../types/pagination';
+
+const createComment = async (req: AuthenticatedRequest, res: Response) => {
+  if (req.params.comment_id) {
+    const comment_exists = await Comment.findOne({
+      where: { id: req.params.post_id },
+    });
+    if (comment_exists !== null) {
+      res.status(400).send({ error: 'Comment already exists' });
+      return;
+    }
+  }
+
+  const { comment, contentType } = req.body;
+  const commentStr = comment;
```

Comment:
What's the point of this?

Reply:
If don't have the line `const commentStr = comment`, I get the following error:
![Screenshot_20220328_133822](https://user-images.githubusercontent.com/43073294/160473784-2ad513ac-3956-4b84-8d2b-68394aa0d87f.png)

Reply:
it's cause you're trying to reassign an assigned variable already